eclass/qt5-build.eclass

```diff
@@ -675,12 +675,6 @@ qt5_base_configure() {
 	"${S}"/configure "${conf[@]}" || die "configure failed"
 
 	popd >/dev/null || die
-
-	if [[ ${QT5_MINOR_VERSION} -ge 8 ]]; then
```

Comment:
Removing this block will break 5.9.1. I'm checking now to see what changed upstream in 5.9.9999

Reply:
Live ebuilds are built out-of-source while release are not which is triggering the failure. So, we should keep this block but move it up up slightly to still operate within ``QT5_BUILD_DIR``.

Reply:
good catch